tools/PI/DevHome.PI/ViewModels/BarWindowViewModel.cs

```diff
@@ -66,6 +68,12 @@ public partial class BarWindowViewModel : ObservableObject
     [ObservableProperty]
     private bool _showingExpandedContent;
 
+    [ObservableProperty]
+    private bool _isAlwaysOnTop = true;
+
+    [ObservableProperty]
+    private PointInt32 _windowPosition;
+
     public BarWindowViewModel()
     {
         _dispatcher = Microsoft.UI.Dispatching.DispatcherQueue.GetForCurrentThread();
```

Comment:
This is going to be null when not running on the UI thread (which this isn't guaranteed to be doing). Use 
```var dispatcherQueue = Application.Current.GetService<DispatcherQueue>();```
or
```var dispatcherQueue = Application.Current.GetService<Window>().DispatcherQueue;```
like here https://github.com/microsoft/devhome/pull/2934
(I'm not actually sure if one is preferred... @nieubank?)

Reply:
Currently, we make sure that BarWindowViewModel is started from a UI thread, but you are right this needs to be updated. I created a new issue fixing this for all of PI: https://github.com/microsoft/devhome/issues/3120

Reply:
both of these are for convenience so I'm not sure either is preferred. We should arguably not have the separate DispatcherQueue accessor and instead always make consumers go through the Window itself which would help if we ever moved more toward a multi-window model.